### PR TITLE
fix: Update takeoff calculator command to highlight new ongoing development

### LINKED ISSUE #495: Update perf command with new information
### Bug type

Wrong data or information

### Describe the bug

Outdated info according to Donbikes. Requests we update the command in the last line to say that we are working on our own takeoff performance calculator.

### Expected behavior

New updated command

### Steps to reproduce

n/a

### References (optional)

Support ops mention - [here](https://discord.com/channels/738864299392630914/838062729398976522/1148145997130760252)

### Additional info (optional)

_No response_

### Discord Username (optional)

valastiri

### PR DESCRIPTION
**Pull request by pdellaert**
## Description

Fixes #495

## Test Results

Screenshot 2023-09-04 at 2 04 10 PM

## Discord Username

straks

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Changelog
 
+Update <small>_ September 2023</small>
+
+- fix: Update takeoff calculator command to highlight new ongoing development (04/09/2023)
```
Comment (Lauvaan) [inline .github/CHANGELOG.md:5]: Technically should probably be lower case, if you want to fix

**Review by Lauvaan — CHANGES_REQUESTED** [review]

**Review by Lauvaan — APPROVED** [review]

**Comment by Valastiri** [comment]
I think we shouldn't remove the existing text. We can put what you've done at the top or bottom fo the command and remove the last line that starts with "This unfortunately [...]". 

So it's clear that the information is hard to come by -- but we are coming up with a solution i.e. in development. 

Thoughts?

**Comment by benw202** [comment]
> I think we shouldn't remove the existing text. We can put what you've done at the top or bottom fo the command and remove the last line that starts with "This unfortunately [...]".
> 
> So it's clear that the information is hard to come by -- but we are coming up with a solution i.e. in development.
> 
> Thoughts?

I also think this would be good

**Comment by benw202** [comment]
Bump

**Comment by pdellaert** [comment]
Updated changes

image

File: src/commands/aircraft/flex-temp.ts [hunk 2]
```diff
@@ -1,10 +1,14 @@
 import { MessageCommandDefinition } from '../../lib/command';
 import { CommandCategory } from '../../constants';
-import { makeEmbed } from '../../lib/embed';
+import { makeEmbed, makeLines } from '../../lib/embed';
 
 const flexEmbed = makeEmbed({
     title: 'FlyByWire A32NX | Flex Temp',
-    description: 'A320neo takeoff performance data is not readily available. To set your Flex Temp in the simulator please see the [Flex Temp Section](https://docs.flybywiresim.com/pilots-corner/beginner-guide/preparing-mcdu/#flex-temp) in our MCDU guide.',
+    description: makeLines([
+        'The Flex Temp can be calculated in the **Developer** version using the Takeoff Performance Calculator in the EFB. Check out the [Documentation](https://docs.flybywiresim.com/flypad-performance/) for more information.',
```
Comment (alepouna) [inline src/commands/aircraft/flex-temp.ts:8]:
```suggestion
        'The Flex Temp can be calculated in the **Development** version using the Takeoff Performance Calculator in the EFB. Check out the [Documentation](https://docs.flybywiresim.com/flypad-performance/) for more information.',
```
or better
```suggestion
        'The Flex Temp can be calculated in the **Development** branch using the Takeoff Performance Calculator in the EFB. Check out the [Documentation](https://docs.flybywiresim.com/flypad-performance/) for more information.',
```

File: src/commands/aircraft/takeoffPerf.ts [hunk 3]
```diff
@@ -3,12 +3,14 @@ import { CommandCategory } from '../../constants';
 import { makeEmbed, makeLines } from '../../lib/embed';
 
 const takeoffPerfEmbed = makeEmbed({
-    title: 'FlyByWire A32NX | Where is the takeoff calculator?',
-    description: makeLines(['Currently, Airbus does not publicly release takeoff performance data for the A320neo, which means it is difficult to create an accurate takeoff calculator.',
+    title: 'FlyByWire A32NX | Takeoff Performance Calculator',
+    description: makeLines([
+        'A Takeoff Performance Calculator is available on the **Developer** version of the A32NX. For more information, check out the [Takeoff Performance Calculator documentation](https://docs.flybywiresim.com/flypad-performance/).',
```
Comment (alepouna) [inline src/commands/aircraft/takeoffPerf.ts:8]: Same as above

File: src/commands/aircraft/takeoffPerf.ts [hunk 4]
```diff
@@ -3,12 +3,14 @@ import { CommandCategory } from '../../constants';
...
         '',
-        'While there are many online calculators available, these often use A320ceo data, and are not accurate.',
+        'For the **Stable** version, see the [V-Speeds Section](https://docs.flybywiresim.com/pilots-corner/beginner-guide/preparing-mcdu/#v-speeds) in our MCDU guide.',
```
Comment (alepouna) [inline src/commands/aircraft/takeoffPerf.ts:10]:
```suggestion
        'For the **Stable** branch, see the [V-Speeds Section](https://docs.flybywiresim.com/pilots-corner/beginner-guide/preparing-mcdu/#v-speeds) in our MCDU guide.',
```

File: src/commands/aircraft/takeoffPerf.ts [hunk 5]
```diff
@@ -3,12 +3,14 @@ import { CommandCategory } from '../../constants';
...
         '',
-        'This unfortunately means that we cannot provide a reliable and accurate takeoff performance calculator at this time.']),
+        '**Note:** The functionality where the LSK next to the V1, VR and V2 speeds can be used to automatically calculate them in the MCDU PERF page has been removed in the Developer version. Use the new Takeoff Performance Calculator instead.',
```
Comment (alepouna) [inline src/commands/aircraft/takeoffPerf.ts:12]: same as above (the abover)

File: src/commands/aircraft/flex-temp.ts (hunk 2 above)
Comment (Sleinmaster) [reply src/commands/aircraft/flex-temp.ts:8]: I would keep it as version.

File: src/commands/aircraft/takeoffPerf.ts (hunk 4 above)
Comment (Sleinmaster) [reply src/commands/aircraft/takeoffPerf.ts:10]: Would leave it as version to keep it in the language of a non Dev.

**Review by alepouna — APPROVED** [review]

**Comment by pdellaert** [comment]
Updated Developer to Development, for "Branch" versus "Version": we use versions everywhere, so I'm keeping those designations.

**Review by ExampleWasTaken — APPROVED** [review]

**Pull request merged**